test/local/senders/index.js

```diff
@@ -252,19 +252,21 @@ describe('lib/senders/index', () => {
             email._ungatedMailer.unblockCodeEmail = sinon.spy(() => P.resolve({}))
             return email.sendUnblockCode(EMAILS, acct, {code: code})
           })
-          .catch(e => {
-            assert.equal(errorBounces.check.callCount, 3)
+          .then(() => {
+            assert.fail('should have blocked the send')
```

Comment:
This test would silently succeed even if no error was thrown; it now explicitly asserts that there was an error.